ghost/core/test/e2e-api/admin/integrations.test.js

```
@@ -348,8 +348,8 @@ describe('Integrations API', function () {
         await request.del(localUtils.API.getApiQuery(`integrations/${createdIntegration.id}/`))
             .set('Origin', config.get('url'))
             .expect(204)
-            .expect((_res) => {
-                _res.body.should.be.empty();
+            .expect((res) => {
```

ESLint

[error] 'res' is already declared in the upper scope on line 337 column 15.

(no-shadow)